client_script.py

```diff
@@ -0,0 +1,41 @@
+
+from SshToServer import SshToServer
+import pandas as pd
+import os
+import json
+from dotenv import load_dotenv
+load_dotenv()
```

Comment:
what is it and why are we using it?

Reply:
this is the module you wrote during the course to interact with the server running a remote command in the main branch under SshToServer.py